backend/apps/slack/management/commands/owasp_match_channels.py

```
@@ -0,0 +1,36 @@
+        ]:
+            content_type = ContentType.objects.get(app_label=app_label, model=model_name)
+            for entity in model.objects.all():
+                # Example: match conversation by name
+                conversations = Conversation.objects.filter(name__icontains=entity.name)
+                for conv in conversations:
+                    obj, was_created = EntityChannel.objects.get_or_create(
+                        content_type=content_type,
+                        object_id=entity.pk,
+                        conversation=conv,
+                        defaults={
+                            'is_main_channel': False,
```

Ruff (0.12.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)